internal/workload/manager.go

```diff
@@ -223,5 +230,17 @@ func (w *WorkloadManager) toPod(workload *models.Workload) (*v1.Pod, error) {
 	}
 	pod.Kind = "Pod"
 	pod.Name = workload.Name
+	exportVolume := volumes.HostPathVolume(w.volumesDir, workload.Name)
```

Comment:
Do we guarantee `workload.Name` to be unique?

Reply:
Yes, it has to be, as it is a name of a CR.

Reply:
The name of a CR is unique only in context of specific namespace. I wonder whether only name is enough.

Reply:
Yes. EdgeDevice and EdgeDeployment have to be in the same namespace as per current design. It is not possible to deploy workloads from different namespaces.